plugins/pipelines/app/decorators/stage_decorator.rb

```diff
@@ -0,0 +1,9 @@
+Stage.class_eval do
+  serialize :next_stage_ids, Array
+
+  def next_stage
```

Comment:
Not useless, if you aren't using this pipeline stuff, then the existing functionality is what you'll get. Which IMO was a pretty useless button in the first place. So yea, I'm probably agreeing with you! :)